### ISSUE TITLE
GD-198: SceneRunner has no `set_property`

### ISSUE THREAD

**Issue by clemens-tolboom** [MikeSchulze/gdUnit4#198]
**The used GdUnit4 version:**

4.1.1 (and master)

**The used Godot version:**

v4.0.3.stable.official [5222a99f5]

**OS including version:**
MacOS Ventura 13.4 (22F66)
OpenGL API 4.1 Metal - 83.1 - Compatibility - Using Device: Apple - Apple M1

**Describe the bug**

I need to set a property of my scene but there is no method `set_property` on runner

One can use `runner._current_scene.x = 9` but that is not as intended.

**Steps to Reproduce**

```gdscript
extends Node

@export var x:int = 10
```

and test

```gdscript
func test_y() -> void:
	var runner :GdUnitSceneRunner = scene_runner("res://x.tscn")

	var x :int = runner.get_property("x")
	assert_int(x).is_equal(10)
	
	runner._current_scene.x = 9
	x = runner.get_property("x")
	assert_int(x).is_equal(9)
	
	runner._current_scene['x'] = 8
	x = runner.get_property("x")
	assert_int(x).is_equal(8)

	runner.set_property("x", 7) # <==== wished for method
	x = runner.get_property("x")
	assert_int(x).is_equal(7)
```
**Minimal reproduction project:**

[issue_198.zip](https://github.com/MikeSchulze/gdUnit4/files/11724466/issue_198.zip)

**Comment by clemens-tolboom** [comment]
See PR #199

**Comment by clemens-tolboom** [comment]
This is now superseded by #202 thus closes #201

See also #200
